src/slide-pane/index.ts

```diff
@@ -305,7 +305,14 @@ export class SlidePane extends I18nMixin(ThemedMixin(WidgetBase))<SlidePanePrope
 											},
 											[
 												closeText,
-												w(Icon, { type: 'closeIcon', theme, classes })
+												w(Icon, {
+													type: 'closeIcon',
+													theme,
+													classes,
+													extraClasses: {
```

Comment:
If there is a better way to add additional classes to a widget please let me know.